misc/nsis/uninstall.nsh

```diff
@@ -178,6 +185,11 @@ Function un.onInit
 
   ${ifnot} ${UAC_IsInnerInstance}
   ${andif} $RunningFromInstaller = 0
+    ${if} ${UAC_IsAdmin}
+    ${andif} $RunningAsUser = 0
+      ${StdUtils.ExecShellAsUser} $0 "$INSTDIR\${UNINSTALL_FILENAME}" "open" "/user $R0"
```

Comment:
Are you sure about that? Because with the uninstaller, `$INSTDIR` contains the directory where the uninstaller's executable is.

Reply:
Yeah when I run the uninstaller from account Unpriv, Admin's local app data folder is in $INSTDIR. Am I maybe doing something wrong? I just figured since it's getting elevated, $INSTDIR switches over to the Admin's local app data

Reply:
Are you by any chance setting `$INSTDIR` manually through registry?

Reply:
Yes I believe I am

Reply:
Then you should set it after the `ExecShellAsUser` command.